git-branchless-revset/src/builtins.rs

```diff
@@ -513,6 +567,19 @@ fn fn_current(ctx: &mut Context, name: &str, args: &[Expr]) -> EvalResult {
     Ok(result.into_iter().collect::<CommitSet>())
 }
 
+#[instrument]
+fn fn_merges(ctx: &mut Context, name: &str, args: &[Expr]) -> EvalResult {
+    eval0(ctx, name, args)?;
+    // "pattern matchers" normally operate with a pattern, but we can make use
+    // of this system to look for merge commits w/o the use of one
+    make_pattern_matcher(
+        ctx,
+        name,
+        args,
+        Box::new(move |_repo, commit| Ok(commit.get_parents().len() > 1)),
```

Comment:
issue: Use `commit.get_parent_count()` here. If you use `get_parents()` directly, then we have to actually load each parent object from disk, even though we know the parent count from just looking at the current commit.